### DISCUSSION TITLE
Anyone having problems with updating website: Can't connect to server?

### DISCUSSION THREAD

**Discussion by IngoBelka** [datenstrom/community#763] [Ask a question] [answered]
Hej,

I'm not shure if it is ab bug or a problem with my configuration.

Today I tried to look for updates for Yellow. Doing this I get the error message: "Can't connect to server 'https://github.com'!". I get this in the browser and on command line as well.

A nslookup and ping are possible without problems. A `wget https://github.com` saves the indexfile of Github with no problem, too.

Does anybody has an idea, what's giong wrong.

My current version is 0.8.19. The update to that version was absolutly fine.

**Comment by IngoBelka** [comment, marked as answer]
OK, I find it out.

Because of the terrible current war in the Ukraine, we got suggestions to block a greater list of URI's to protect our networks. One of these is a subdomain of Github.

Problem is solved therefor.

**Reply by markseuffert** [reply]
Thanks for your feedback. The updates are hosted on GitHub, downloading raw data from `https://github.com/` is in reality coming from `https://raw.githubusercontent.com`. Make sure these URLs are not blocked, see also #467.
